packages/grafana-data/package.json

```diff
@@ -74,5 +74,9 @@
     "sinon": "13.0.1",
     "tinycolor2": "1.4.2",
     "typescript": "4.6.4"
+  },
+  "peerDependencies": {
```

Comment:
Does this mean that all plugins that declare`@grafana/data` or `@grafana/ui` as a dependency will now require `dependency` declarations for `react` and `react-dom` in their `package.json`?

Reply:
Yes, but actually it is a pretty common way of working: any react library you use shouldn't force you to use a specific react version. Anyway, if you foresee any problems with this change, I'll close the PR, it is not mandatory for us